packages/assistant/src/RPA/Assistant/flet_client.py

```diff
@@ -0,0 +1,183 @@
+import atexit
+import logging
+import os
+import signal
+from typing import Callable, List, Optional, Union, Literal, Tuple
+
+import flet
+from flet import Control, Page, ScrollMode
+from flet.utils import is_windows
+
+from RPA.Assistant.types import Result, Location
+
+
+def resolve_absolute_position(
+    location: Union[Location, Tuple],
+) -> Tuple[int, int]:
+
+    if location is Location.TopLeft:
+        left_coordinate = 0
+        top_coordinate = 0
+        return (left_coordinate, top_coordinate)
+    elif isinstance(location, tuple):
+        return location
+    else:
+        raise ValueError("Locations other than Center or TopLeft are not yet supported")
+
+
+class FletEvent:
+    target: str
+    name: str
+    data: str
+    control: Control
+    page: Page
+
+
+class FletClient:
+    """Class for wrapping flet operations"""
+
+    def __init__(self) -> None:
+        self._conn = self._preload_flet()
+        self.elements: List[List[Control]] = [[]]
```

Comment:
On a quick skim: Why the 2d array? What is a list of controls? An element?

Reply:
fixed in 5cfb75d6, this was not anymore used so the 2d-ness has now been removed.